server/lobbyconnection.py

```diff
@@ -216,6 +216,15 @@ async def on_message_received(self, message):
                 self.get_user_identifier(),
                 cmd
             )
+        except OperationalError:
+            # When the database goes down, SqlAlchemy will throw an OperationalError
+            await self.send({
+                "command": "notice",
+                "style": "error",
+                "text": "Unable to connect to database. Please try again later."
+            })
+            # Make sure to abort here to avoid a thundering herd problem.
```

Comment:
I think we need to use the `kick` style to prevent the client from trying to automatically reconnect. @Sheikah45?

Reply:
I was thinking that's what https://github.com/FAForever/server/pull/1009/files#diff-de94292bdcec0d8f98b8eb76040f97830866bed0c1fb5b55165a9e54f175ea0bR227 would do for the client, but if that is not the case, we for sure need to send the client a "Stop and don't retry" message.

Reply:
Yea, there's currently no good system for telling the client that it should not attempt to reconnect. I believe the `kick` style message is all we have. It might be cleaner to add a dedicated message for that but that would be for another PR and also require client changes.

Reply:
I think https://github.com/FAForever/downlords-faf-client/blob/ca8554d8767e748498dee37caa39cf3f657284a6/src/main/java/com/faforever/client/remote/FafServerAccessor.java#L115 could be an issue since the logic is if we were connected and suddenly were disconnected, retry the login and connect. Net new connections should be okay since it will be going from connecting to disconnected.

Reply:
The main problem with the kick style is that it will also close the client. If this is desired behavior then sure but otherwise it shouldn't be kick.

Reply:
@Sheikah45 @Askaholic I think this should work for what we want then. This follows the same pattern as if someone were banned where it sends a message and aborts the connection.

Reply:
Sounds good, we'll leave it like this then.